imports/plugins/core/orders/client/templates/orders.js

```diff
@@ -196,6 +210,9 @@ Template.ordersListItem.events({
     if (order.workflow.status === "new") {
```

Comment:
This will also save you from multiple `if blocks`

```
const status = {
    new: "processing",
    cancelled: "cancelled"
}

Meteor.call("workflow/pushOrderWorkflow", "coreOrderWorkflow", status[order.workflow.status], order);
```